CHANGELOG.md

```diff
@@ -212,9 +212,8 @@
   [#8544](https://github.com/Kong/kong/pull/8544)
   [#8752](https://github.com/Kong/kong/pull/8752)
   [#8994](https://github.com/Kong/kong/pull/8994)
-- Bumped resty.openssl from 0.8.5 to 0.8.7
-  [#8592](https://github.com/Kong/kong/pull/8592)
-  [#8753](https://github.com/Kong/kong/pull/8753)
+- Bumped resty.openssl from 0.8.8 to 0.8.10
```

Comment:
0.8.5 to 0.8.10, also leve previous pr links there.

Reply:
@bungle, previous entries added back.